model-optimizer/unit_tests/mo/graph/graph_test.py

```diff
@@ -429,6 +429,79 @@ class TestNewGraphAPIMiddle(unittest.TestCase):
         'const_1_data': {'value': None, 'shape': None, 'kind': 'data'},
     }
 
+    nodes_10_in_10_out = {
+        'in_0': {'type': 'Parameter', 'value': None, 'kind': 'op', 'op': 'Parameter'},
+        'in_0_data': {'value': None, 'shape': None, 'kind': 'data'},
```

Comment:
please generate this dictionary of nodes. Now it looks overloading with similar code

Reply:
Done.